### ISSUE TITLE
Cannot add members to Bridge namespace.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1442]
### Expected
```
Hello World
```
### Actual
```
C# to JavaScript Compilation Failed

Bridge.Contract.EmitterException: Namespace 'Bridge' uses reserved name 'Bridge'.
This name is reserved for Bridge.NET core.
   at Bridge.Translator.Inspector.ValidateNamespace(NamespaceDeclaration nsDecl)
   at Bridge.Translator.Inspector.VisitNamespaceDeclaration(NamespaceDeclaration namespaceDeclaration)
   at ICSharpCode.NRefactory.CSharp.NamespaceDeclaration.AcceptVisitor(IAstVisitor visitor)
   at Bridge.Contract.Helpers.AcceptChildren(AstNode node, IAstVisitor visitor)
   at Bridge.Translator.Inspector.VisitSyntaxTree(SyntaxTree node)
   at Bridge.Translator.Translator.InspectTypes(MemberResolver resolver, IAssemblyInfo config)
   at Bridge.Translator.Translator.Translate()
   at Live.LiveTranslator.Translate(String& output)
   at Live.TranslateHandler.Translate(String csCode)
```
### Steps To Reproduce
[Live Bridge](http://live.bridge.net/#cd29116b853c84a627363163c65c29e2)
[Dot Net Fiddle](https://dotnetfiddle.net/7eOlMk)
```csharp
using Bridge.Html5;
using System;

namespace Bridge
{
    public static class App
    {
        [Ready]
        public static void Main ()
        {
            Console.WriteLine("Hello World");
        }
    }
}
```
You can add to the namespace (s): Microsoft and System.
So, why not Bridge.

**Comment by LeonidVeriga** [comment]
This is related to #88 and the [forum thread](http://forums.bridge.net/forum/community/help/55-installation-problem-in-vs-2013), in particular this post:
> This problem was caused by the Visual Studio Project name being selected as Bridge, which created a Bridge.dll, which caused a collision with the actual Bridge.dll.

> The solution was to rename the Visual Project to something other than Bridge.

> We're looking into a way to work around this naming conflict automatically, although for now using "Bridge" as a project name will cause a collision and build Exception.

**Comment by LeonidVeriga** [comment]
There is a new config [enableReservedNamespaces](https://github.com/bridgedotnet/Bridge/issues/1656) implemented in **15.0**. Setting to `true` allows adding members into `Bridge` namespace:
```js
{
  "assembly": {
    "enableReservedNamespaces": [false]|true,
  }    
}
```
